### PR TITLE
Update PMT Grammar and Explanation

### PR DESCRIPTION
**Pull request by ckartik**
* Adds small grammar corrections
* Adds some additional context for the Merkle root concept
* Makes a small change around word choice. Using integrity instead of authentication to explain the integrity guarantee provided by Merkle tree representation of data.

### CODE REVIEW

**Comment by Ameralameri** [comment]
issuanceDate": null, 	"expiryDate": null, 	"stamps": [ 		{ 			"provider": "FirstEthTxnProvider", 			"credential": { 				"type": [ 					"VerifiableCredential" 				], 				"proof": { 					"jws": "eyJhbGciOiJFZERTQSIsImNyaXQiOlsiYjY0Il0sImI2NCI6ZmFsc2V9..BjXv0mrLCgYeq94RugkpIvODfRBL1_n2XtaHafgruHnLgi2N0O709gelKBdJpPKqOd5ejKd69LuCFEeMyvbJAQ", 					"type": "Ed25519Signature2018", 					"created": "2023-06-11T16:12:45.010Z", 					"proofPurpose": "assertionMethod", 					"verificationMethod": "did:key:z6MkghvGHLobLEdj1bgRLhS4LPGJAvbMA1tn2zcRyqmYU5LC#z6MkghvGHLobLEdj1bgRLhS4LPGJAvbMA1tn2zcRyqmYU5LC" 				}, 				"issuer": "did:key:z6MkghvGHLobLEdj1bgRLhS4LPGJAvbMA1tn2zcRyqmYU5LC", 				"@context": [ 					"https://www.w3.org/2018/credentials/v1" 				], 				"issuanceDate": "2023-06-11T16:12:45.009Z", 				"expirationDate": "2023-09-09T16:12:45.009Z", 				"credentialSubject": { 					"id": "did:pkh:eip155:1:0x20fE9F5896A3021Ca6eA78005a15AE954dC134E6", 					"hash": "v0.0.0:2XE4XEPRWXL4JfVpFk8Sd4K1GCVJwN90icmGzyzLKOM=", 					"@context": [ 						{ 							"hash": "https://schema.org/Text", 							"provider": "https://schema.org/Text" 						} 					], 					"provider": "FirstEthTxnProvider" 				} 			} 		}, 		{ 			"provider": "EthGTEOneTxnProvider", 			"credential": { 				"type": [ 					"VerifiableCredential" 				], 				"proof": { 					"jws": "eyJhbGciOiJFZERTQSIsImNyaXQiOlsiYjY0Il0sImI2NCI6ZmFsc2V9..mfAF0vsAh1SIcZ6Hc50wBs030ldvGCB0pVdNAsC3wRhK97q58mKG5ULDC7Wt7BTQfE510nEQk5k-TifoBWxcAw", 					"type": "Ed25519Signature2018", 					"created": "2023-06-11T16:12:45.265Z", 					"proofPurpose": "assertionMethod", 					"verificationMethod": "did:key:z6MkghvGHLobLEdj1bgRLhS4LPGJAvbMA1tn2zcRyqmYU5LC#z6MkghvGHLobLEdj1bgRLhS4LPGJAvbMA1tn2zcRyqmYU5LC" 				}, 				"issuer": "did:key:z6MkghvGHLobLEdj1bgRLhS4LPGJAvbMA1tn2zcRyqmYU5LC", 				"@context": [ 					"https://www.w3.org/2018/credentials/v1" 				], 				"issuanceDate": "2023-06-11T16:12:45.264Z", 				"expirationDate": "2023-09-09T16:12:45.264Z", 				"credentialSubject": { 					"id": "did:pkh:eip155:1:0x20fE9F5896A3021Ca6eA78005a15AE954dC134E6", 					"hash": "v0.0.0:4Ld88Biy+AdKw6Gd35PyfvdCGQcwadk34mL7ygJYfVE=", 					"@context": [ 						{ 							"hash": "https://schema.org/Text", 							"provider": "https://schema.org/Text" 						} 					], 					"provider": "EthGTEOneTxnProvider" 				} 			} 		}

**Review by minimalsm — APPROVED** [review]
LGTM, @jmcook1186 could you have a quick look 🙏

File: src/content/developers/docs/data-structures-and-encoding/patricia-merkle-trie/index.md [hunk 1]
```diff
@@ -62,9 +62,9 @@ The update and delete operations for radix tries can be defined as follows:
                 return hash(newnode)
 ```
 
-A "Merkle" Radix tree is built by linking nodes using deterministically-generated cryptographic hash digests. This content-addressing (in the key/value DB `key == keccak256(rlp(value))`) provides cryptographic authentication of the stored data. If the root hash of a given trie is publicly known, then anyone can provide a proof that the trie includes a given value at a specific path by providing the hashes of each node joining a specific value to the tree root.
+A "Merkle" Radix tree is built by linking nodes using deterministically-generated cryptographic hash digests. This content-addressing (in the key/value DB `key == keccak256(rlp(value))`) provides a cryptographic integerity guarantee of the stored data. If the root hash of a given trie is publicly known, then anyone with access to the underlying leaf data can construct a proof that the trie includes a given value at a specific path by providing the hashes of each node joining a specific value to the tree root.
```
Comment (jmcook1186) [inline src/content/developers/docs/data-structures-and-encoding/patricia-merkle-trie/index.md:65]:
```suggestion
A "Merkle" Radix tree is built by linking nodes using deterministically-generated cryptographic hash digests. This content-addressing (in the key/value DB `key == keccak256(rlp(value))`) provides a cryptographic integrity guarantee of the stored data. If the root hash of a given trie is publicly known, then anyone with access to the underlying leaf data can construct a proof that the trie includes a given value at a specific path by providing the hashes of each node joining a specific value to the tree root.
```

**Review by jmcook1186 — APPROVED** [review]
fixed typo, now lgtm!

**Comment by minimalsm** [comment]
@all-contributors please add @ckartik for content

**Pull request merged**